web/styles/typography.sass

```diff
@@ -60,14 +60,14 @@ p
     font-size: 24px
     line-height: 36px
     margin-bottom: 40px
-    font-style: bold
+    font-weight: bold
```

Comment:
not sure if this is the cause but the headers don't seem to be bolded anymore:

![image](https://user-images.githubusercontent.com/21287547/46634018-6cc5a500-cb04-11e8-9153-cc5c5a5a008b.png)

Reply:
What browser is this? On my end it looks bold on both Chrome and Firefox

![image](https://user-images.githubusercontent.com/12876696/46634387-9b904b00-cb05-11e8-91a9-cdac9d2eec5f.png)

Reply:
oh it was on safari LOL